### PR TITLE
multiplicityPbPb.cxx (eta and zvtx cuts)

### PR DESCRIPTION
**Pull request by hhesouno**

### CODE REVIEW

File: PWGMM/Mult/Tasks/CMakeLists.txt [hunk 1]
```diff
@@ -60,6 +60,8 @@ o2physics_add_dpl_workflow(heavy-ion-mult
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
                     COMPONENT_NAME Analysis)
 
+
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/CMakeLists.txt:63]: Please avoid adding extra lines in unrelated files, check with `git diff` before committing.

File: PWGMM/Mult/Tasks/multiplicityPbPb.cxx [hunk 2]
```diff
@@ -123,9 +130,14 @@ struct multiplicityPbPb {
 
       histos.fill(HIST("EtaZvtxTracks"), track.eta(), collision.posZ());
       histos.fill(HIST("PhiEtaTracks"), track.phi(), track.eta());
+
+      if(track.eta()<0.1) ++trackCounter_01;
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/multiplicityPbPb.cxx:134]: I think it is supposed to be `std::abs(track.eta()) < 0.1`?

**Review by aalkin — CHANGES_REQUESTED** [review]

File: PWGMM/Mult/Tasks/CMakeLists.txt (hunk 1 above)
Comment (hhesouno) [reply PWGMM/Mult/Tasks/CMakeLists.txt:63]: I am sorry, I do not know where is that coming from. I did not modify that file intentionally.

File: PWGMM/Mult/Tasks/multiplicityPbPb.cxx (hunk 2 above)
Comment (hhesouno) [reply PWGMM/Mult/Tasks/multiplicityPbPb.cxx:134]: Yes, thank you so much!
